Documentation/k8s-block.md

```diff
@@ -26,7 +26,7 @@ metadata:
   namespace: rook
 spec:
   replicated:
-    size: 1
+    size: 3
```

Comment:
using replication of 3 by default will break single node deployments, such as minikube.  those are more developer scenarios though, and I'm beginning to think there may be a separation of audiences for our [Documentation](https://github.com/rook/rook/tree/master/Documentation) dir and our [cluster/examples/kubernetes](https://github.com/rook/rook/tree/master/cluster/examples/kubernetes) directory.

I think the raw yaml files in `cluster/examples/kubernetes` should stay compatible with VERY small (single node) dev clusters so devs don't constantly have to change them from their committed values during their workflow.

The samples in the `Documentation` dir however have a broader audience of users that have real (bigger) clusters and updating those (like you've done here) to actually have redundancy turned on by default is probably the right idea.

Reply:
I've done the changes in cluster/examples/kubernetes and filesystem CRD documentation accordingly.

About replicated one, it's now homogeneous on documentation side (set to 3) but not on k8s manifests one (there is used 1,2 and 3) :

```
➜  rook git:(pool-documentation) ag replicated: -A 1 Documentation         
Documentation/object-store-crd.md
22:    replicated:
23-      size: 3

Documentation/k8s-block.md
28:  replicated:
29-    size: 3

Documentation/pool-crd.md
21:  replicated:
22-  #  size: 3

Documentation/k8s-filesystem.md
31:    replicated:
32-      size: 3

Documentation/k8s-object.md
28:    replicated:
29-      size: 3

Documentation/filesystem-crd.md
22:    replicated:
23-      size: 3
➜  rook git:(pool-documentation) ag replicated: -A 1 cluster               
cluster/examples/kubernetes/1.6/rook-object.yaml
10:    replicated:
11-      size: 2

cluster/examples/kubernetes/rook-object.yaml
11:    replicated:
12-      size: 3

cluster/examples/kubernetes/rook-pool.yaml
10:  replicated:
11-    size: 1

cluster/examples/kubernetes/rook-filesystem.yaml
9:    replicated:
10-      size: 3

cluster/examples/kubernetes/1.5/rook-storageclass.yaml
7:  replicated:
8-    size: 1

cluster/examples/kubernetes/rook-storageclass.yaml
7:  replicated:
8-    size: 1
➜  rook git:(pool-documentation) 
```
We could set them all to 3 and let single node users practice their "sed-fu" in order to bring that back to 1 or we could go the "works for all" way and set it to 1 (and then the same for erasureCoded 1 data chunk and 0 coding chunk ? would that even works ?) at the price of no data safety.
What do you think about it ? I would probably go the cluster way and set it to 3 and let single node users do the replacement.

Reply:
@jbguerraz with erasure coding it is required to set at least 2 data chunks and 1 coding chunk. @jbw976 agreed that sounds like a good approach for all the examples in the documentation to be with redundancy for a reasonable 3-node cluster, while the yaml examples would be good to work by default for the most part in a 1-node cluster.

Reply:
i'm actually okay with the way the `cluster/examples/kubernetes` replicated settings are now then with no further modifications (even if they are heterogeneous).  I think the update you've made to ensure all the EC values are reasonable for small clusters was the most important part.